roles/openshift_certificate_expiry/test/test_load_and_handle_cert.py

```diff
@@ -0,0 +1,68 @@
+'''
+ Unit tests for the FakeOpenSSL classes
```

Comment:
@detiber I'm sure you already noticed, but this is the only thing I spotted in this commit that needs changing.